playground/list-password-profiles.ts

```diff
@@ -0,0 +1,33 @@
+import * as moltin from '..'
+
+async function main() {
+  const gateway = moltin.gateway({
+    host: 'epcc-integration.global.ssl.fastly.net',
+    client_id: 'R7paB5xa2ngzuEM8N6PkTOGpO9jTcAgW1DO2U2KzOl',
+    client_secret: 'lsKTS0hKMzbxpNBAVjX0NoXt8FeALVPcQxh4lgofWD',
```

Comment:
No need to commit playground files. Especially not the ones containing secrets :)